src/test/java/org/mskcc/cmo/metadb/ValidRequestCheckerTest.java

```diff
@@ -0,0 +1,112 @@
+package org.mskcc.cmo.metadb;
+
+import java.util.Map;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mskcc.cmo.metadb.config.MockDataConfig;
+import org.mskcc.cmo.metadb.model.MockJsonTestData;
+import org.mskcc.cmo.metadb.service.ValidRequestChecker;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import junit.framework.Assert;
+
+@ContextConfiguration(classes = MockDataConfig.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+@ComponentScan("org.mskcc.cmo.metadb.service")
+public class ValidRequestCheckerTest {
+    
```

Comment:
did you add any new test jsons? I haven't checked myself but I would like to confirm that there's at least one test json where the sample metadata is missing all of these.

In these cases you prob don't need an entire mocked request json. Just a JSON with the sample metadata would be fine that you can test the individual `ValidRequestCheckerImpl` methods that are evaluating the `sampleMap` values.

Reply:
maybe i can a request with just one sample with either null or empty fields